README.md

```diff
@@ -108,6 +135,40 @@ console.log("Thing status",thingStatusInfo.getStatus());
 
 See [openhab-js](https://openhab.github.io/openhab-js) for a complete list of functionality
 
+### Event Object
+
+**NOTE**: Note that `event` object is different in UI based rules and File based rules! This section is only valid for UI based rules.
+
+When you use "Item event" as trigger (i.e. "[item] received a command", "[item] was updated", "[item] changed"), there is additional context available for the action in a variable called `event`.
+
+This tables gives an overview over the `event` object for most common trigger types:
+
+| Property Name  | Type                                                                                                                 | Trigger Types                          | Description                                                                                                   |                        |     |

```

Comment:
Can please remove that last empty column?

Reply:
https://github.com/openhab/openhab-js/pull/110/commits/9315b918b593c1eca955ec1f6ddbc37e6843733e